lib/ExternalModule.js

```diff
@@ -27,6 +27,10 @@ class ExternalModule extends Module {
 	}
 
 	chunkCondition(chunk) {
+		if (this.externalType === "amd") {
```

Comment:
Is it possible to also support `umd` here too?

Reply:
UMD support should be possible via a similar pattern- however I am still hoping to get some feedback from someone on the team before putting more work into this approach.

Reply:
For sure, would be good to get input from @sokra or @TheLarkInn on this. The benefits for us UMD/AMD users is huge!